hw1/01-flexbox.md

```diff
@@ -2,6 +2,25 @@
 
```

Comment:
an additional line between each items line would be better. and each line should be a single piece of statements.

Reply:
I'll update that! Thanks!